src/main/java/edu/ucsb/cs156/example/documents/FeatureProperties.java

```diff
@@ -0,0 +1,45 @@
+package edu.ucsb.cs156.example.documents;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FeatureProperties {
+    @Id
+    private String _id;
+
+    private float mag;
+    private String place;
+    private int time;
+    private int updated;
+    private String url;
+    private String detail;
+    private int felt;
+    private int cdi;
```

Comment:
Seems like the cdi should be a float value (as per the second example given)

Reply:
Fixed ty!